### DISCUSSION TITLE
Seven-bit cron expressions

### DISCUSSION THREAD

**Discussion by Leowuqunqun** [Hexagon/croner#209] [Ideas]
Whether seven-bit cron expressions are supported ？ 
 example: 0 0 10 * * ? *

**Comment by Hexagon** [comment]
Hello! No, seven part patterns are not supported, what use case do you have for this?

**Comment by Leowuqunqun** [comment]
0 0 10 * * ? * 
This is my case, when someone knows that cron supports 7-bit they try it, when using other frameworks they all support parsing of 7-bit expressions, although the last bit can only be supported as a *, this expression if not consistent and more headaches

**Comment by Hexagon** [comment]
Converting this to a discussion, seven part cron expressions are not a standard, but could be useful. I suppose the seventh part refer to years?

**Reply by Leowuqunqun** [reply]
Yes, the seventh is the year
